### DISCUSSION TITLE
Add shared link to email reports sent to clients

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#869] [Anything Else]
"It would be great if there was an option to include a dashboard link in the weekly/monthly emails which clients receive. Currently it just shows the stats.

Perhaps the links set in Visibility > Shared links could be numbered - and there could be a dropdown option next to the recipient email address where you could select a “Shared link” to include in the email."

(feedback via email)

**Comment by ukutaht** [comment]
Yes this makes sense.

> Perhaps the links set in Visibility > Shared links could be numbered - and there could be a dropdown option next to the recipient email address where you could select a “Shared link” to include in the email."

I think we have another request to add labels or names to shared links. Instead of numbers we could use those names once implemented
